snippets/card-product.liquid

```diff
@@ -11,7 +11,8 @@
   - lazy_load: {Boolean} Image should be lazy loaded. Default: true (optional)
   - show_quick_add: {Boolean} Show the quick add button.
   - section_id: {String} The ID of the section that contains this card.
-  - horizontal_class: {Boolean} Add a card--horizontal class if set to true. Default: false
+  - horizontal_class: {Boolean} Add a card--horizontal class if set to true. Default: false (optional)
+  - horizontal_add_to_cart: {Boolean} Changes the ATC button styles when set to true. Default: false (optional)
```

Comment:
I wonder if we can rename this to just `add_to_cart`? 🤔 Is there a reason to keep the horizontal? Or maybe it could be `complementary_add_tocart` and `complementary_class`

No strong opinion, but I find the name a bit confusing

Reply:
Yeah I like it 👍 Thanks for those suggestions 🙂